tests/yam/agama/boot_encrypted.pm

```diff
@@ -0,0 +1,44 @@
+## Copyright 2025 SUSE LLC
```

Comment:
name of file: boot_encrypted_partitions.pm

Reply:
updated